.github/workflows/tests.yml

```diff
@@ -23,6 +25,6 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install tox tox-gh-actions
```

Comment:
Let me ping @redtoad in case he has any comments, since he's the one who originally added the Github Actions config.